src/host_capabilities/verification.rs

```diff
@@ -1,11 +1,21 @@
 use anyhow::{anyhow, Result};
-use serde::{Deserialize, Serialize};
+use serde::{Deserialize, Deserializer, Serialize};
 use std::collections::HashMap;
+use url::Url;
 
 use crate::host_capabilities::CallbackRequestType;
 
-/// TODO: write docs
-pub fn verify_image(image: &str, config: Config) -> Result<bool> {
+type WapcVerifyFN = fn(&[u8]) -> Result<Vec<u8>>;
+
+pub fn verify_image(image: &str, config: LatestVerificationConfig) -> Result<bool> {
```

Comment:
We should document this public function, noting that the function expects LatestVerificationConfig to be sanitized (eg UrlPrefix), which happens on deserialization. If not, one must sanitize it manually.